# Request
float64[] joint_angles
---
# Response
geometry_msgs/Point foot_position
